packages/e2e-tests/src/specs/editor/media/insertWebMVideo.js

```diff
@@ -22,9 +23,12 @@ import {
   insertStoryTitle,
   uploadMedia,
   deleteMedia,
+  withUser,
 } from '@web-stories-wp/e2e-test-utils';
 
 describe('Inserting WebM Video', () => {
+  withUser('admin', 'password');
```

Comment:
Why do we need these anywhere now?

Reply:
Theoretically they shouldn't be needed, as `bootstrap.js` already does this. Added to every test for extra safety though.

I can try to see whether it can be safely removed again though. Would make the tests a bit cleaner 👍